### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: lunchbox/CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,6 @@
 set(LUNCHBOX_PUBLIC_HEADERS
   algorithm.h
   any.h
```
Comment (tribal-tec) [inline lunchbox/CMakeLists.txt]: remove any as well?

**Comment by tribal-tec** [comment]
+1 otherwise

File: lunchbox/CMakeLists.txt (hunk 1 above)
Comment (eile) [inline lunchbox/CMakeLists.txt]: Still used by dash::Attribute.

**Comment by eile** [comment]
Damn, missed the edit. @hernando Apologies, I guess I did test compile on OS X. Feel free to revert if it bothers you, otherwise I can look into it tomorrow.

**Comment by hernando** [comment]
I don't remember the details, but RTNeuron won't compile without this.

**Comment by tribal-tec** [comment]
Another option would be to move any to RTNeuron?

**Comment by hernando** [comment]
It's used in the serialization of AttributeMap objects. Actually the AttributeMap and all its functionalities (Python wrapping, serialization, callbacks, ...) should move into a separate library one day, then it would make sense to move the any serialization there.  For the moment I'm not sure, I'll leave the decision to you.

**Pull request merged**